### PR TITLE
Milestone 1

### PR DESCRIPTION
**Pull request by tomdan-ai**
### **On this pull request the following requirements were made:**

- [ ] I set up linter configurations.
- [ ] I ignored node modules.
- [ ] I created HTML with an h1 tag: Hello microverse.
- [ ] I styled the heading with CSS.
- [ ] I edited the README.md file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,19 +56,19 @@ Live Server
...
 👤 **Author1**
 
-- GitHub: [Gmanlove](https://github.com/Gmanlove)
+- GitHub: [tomdan-ai](https://github.com/tomdan-ai)
 
 
 ## 🙏 Acknowledgments <a name="acknowledgements"></a>
 
-I would like to thank Stephen Okhaishie
+I would like to thank Emmnauel Obi
```
Comment (Gambit142) [inline README.md:85]: - [ ] Kindly include 1-3 features of this project in the **_Key Features_** section of your README.md file. You can include learning how to setup linters, learning how to git ignore files and so on. 👍

File: README.md [hunk 2]
```diff
@@ -56,19 +56,19 @@ Live Server
...
-git@github.com:Gmanlove/Hello-Microverse.git
+https://github.com/tomdan-ai/Hello-Microverse.git
 
 
 ## 👥 Authors <a name="authors"></a>
 
 👤 **Author1**
 
-- GitHub: [Gmanlove](https://github.com/Gmanlove)
+- GitHub: [tomdan-ai](https://github.com/tomdan-ai)
```
Comment (Gambit142) [inline README.md:80]: - [ ] Kindly include 1-3 features that can be implemented in the **_Future Features_** section. Features like building a webpage, navigation or whatever you desire. 👍

File: README.md [hunk 3]
```diff
@@ -56,19 +56,19 @@ Live Server
 ### Setup
 
 Clone this repository to your desired folder: 
-git@github.com:Gmanlove/Hello-Microverse.git
+https://github.com/tomdan-ai/Hello-Microverse.git
```
Comment (Gambit142) [inline README.md:73]: - [ ] In your MIT.md file, kindly replace the placeholder `[YOUR NAME]` with your name and every occurrence of `[APP TYPE]` with the name of your application. This makes your LICENSE personalized and defined 😎 👍

**Review by Gambit142 — CHANGES_REQUESTED** [review]
Hi @tomdan-ai 👋, 

Good job so far! You did well for your first project 👏 

## Highlights
- All linters checks passed ✔️ 
- `node_modules` and the `test.md` file were git ignored ✔️ 
- Good commit messages ✔️ 
- Descriptive pull request summary ✔️ 

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:

N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### Please, remember to tag me in your question so I can receive the notification. You can tag me by typing @Gambit142

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by Lizdev-05 — APPROVED** [review]
Hi @tomdan-ai, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- [ ] All linters checks passed ✔️
- [ ] node_modules and the test.md file was git ignored ✔️
- [ ] Good commit messages ✔️
- [ ] Descriptive pull request summary ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**